src/caselawclient/models/identifiers/__init__.py

```diff
@@ -101,6 +108,11 @@ def as_xml_tree(self) -> etree._Element:
     def url_slug(self) -> str:
         return self.schema.compile_identifier_url_slug(self.value)
 
+    @property
```

Comment:
This does mean that an underspecified identifier will score highly... no other default seems sensible, though.